tests/primer/packages_to_lint.json

```diff
@@ -0,0 +1,53 @@
+{
+  "scikit-learn": {
+    "url": "https://github.com/scikit-learn/scikit-learn.git",
+    "branch": "main",
+    "directories": "sklearn"
+  },
+  "flask": {
+    "url": "https://github.com/pallets/flask.git",
+    "branch": "main",
+    "directories": "src/flask"
+  },
+  "keras": {
+    "url": "https://github.com/keras-team/keras.git",
+    "branch": "master",
+    "directories": "keras"
+  },
+  "sentry": {
+    "url": "https://github.com/getsentry/sentry.git",
+    "branch": "master",
+    "directories": "src tests"
```

Comment:
Do we really want to run over tests? That inflates the time to run considerably. We decided to remove `numpy` for the same reason.

Reply:
But do we want lots of project or simply a lot of code ? What if there is a crash when analyzing the tests but not the package itself ? Sure it's taking longer but isn't that also the case for other repositories we could add to have more diversified code ?

Reply:
I think we want some tests, but taking our own tests as an example it is a lot (and really a lot) of the same. In the list with my changes there are still some repositories with tests inside their main directory, but the ones I highlighted have extensive test suites. Inferring a 1000 asserts doesn't provide much extra data to us, so we'll need to find a balance somewhere.
We could also include 1 large project with different sorts of tests, but I think 
checking a lot of asserts doesn't have a good time/value ratio.

Reply:
Fair enough, it's true that tests are more repetitive.

Reply:
f0be9abd235658eec8cf7e89268f908383d75616